earth_enterprise/src/fusion/autoingest/sysman/InsetTilespaceIndex_test.cpp

```diff
@@ -0,0 +1,214 @@
+//
```

Comment:
Minor: it looks like this test still uses the "old" version of the "new" algorithm, i.e., assume images overlap if their quad-tree paths are completely the same. I'm sure you're aware of that; just thought I'd point it out for completeness.

Reply:
Yeah, good catch. I originally removed this file from the SCons build just to kind of get it out of the way while I worked on the new algorithm. I mostly forgot about it after that. Honestly at this point, I'm not even sure this is necessary. The guts of this are being tested by `QuadKeyTree_unittest.cpp `.
Any objection to just removing this file?

Reply:
Removed it.